### ISSUE TITLE
Release on PyPi

### ISSUE THREAD

**Issue by Gagaro** [krisys/django-error-email-throttle#1]
Hi,

Do you plan on releasing this package on PyPi?

Thanks

**Comment by krisys** [comment]
I am planning to do that  @Gagaro . Do you plan on using this app anytime soon?

**Comment by Gagaro** [comment]
Yes I plan to try it out in production really soon. I tried in on a test server and this didn't work though so I have to do some more testing but this feature interests me (I have several servers who went out of hand and produced a few thousands errors in a few minutes).
